### PR TITLE
Add basic documentation

### LINKED ISSUE #20: Add basic documentation

### PR DESCRIPTION
**Pull request by IvayloPapazov**
close #20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,134 @@
+[releases]: https://github.com/aeternity/aeternity/releases
```
Comment (venimus) [inline README.md]: These references usually stay at the bottom of the file

File: README.md [hunk 2]
```diff
@@ -1,20 +1,134 @@
...
+
+```bash
+curl -O https://download.libsodium.org/libsodium/releases/libsodium-1.0.16.tar.gz
+tar -xf libsodium-1.0.16.tar.gz && cd libsodium-1.0.16
+./configure && make && sudo make install && sudo ldconfig
+```
+
+### macOS package
+
+Easiest way to install dependencies is using [Homebrew](https://brew.sh/):
```
Comment (venimus) [inline README.md]: You can add https://brew.sh/ in references as it is repeated at multiple places

File: README.md (hunk 1 above)
Comment (IvayloPapazov) [reply README.md]: 44f4c4d

File: README.md (hunk 2 above)
Comment (IvayloPapazov) [reply README.md]: 44f4c4d

File: README.md [hunk 3]
```diff
@@ -5,16 +5,57 @@ Run below command to install latest version of aeternity node:
...
+* Ubuntu 18.04 LTS (x86-64)
+* macOS High Sierra 10.13 (x86-64)
+* macOS Mojave 10.14 (x86-64)
+* macOS Catalina 10.15 (x86-64)
+
+## Dependencies that will be installed automatically
+
+* [Libsodium](https://download.libsodium.org/doc/)
+* [Openssl](https://www.openssl.org)
+* gmp
```
Comment (dincho) [inline README.md]: [GMP lib](https://gmplib.org)

File: README.md [hunk 4]
```diff
@@ -5,16 +5,57 @@ Run below command to install latest version of aeternity node:
...
+
+See the documentation [here](https://github.com/aeternity/aeternity/blob/master/README.md) detailed instructions for manual installation.
 
 # Run install tests
 
+Downloading the [Git repository](https://github.com/aeternity/installer) itself and:
```
Comment (dincho) [inline README.md]: I don't think this is necessary as it implies the developer is already working with this repo source.
I mean, this line is for developers of the tool, not users.

File: README.md [hunk 5]
```diff
@@ -5,16 +5,57 @@ Run below command to install latest version of aeternity node:
...
+* macOS Mojave 10.14 (x86-64)
+* macOS Catalina 10.15 (x86-64)
+
+## Dependencies that will be installed automatically
+
+* [Libsodium](https://download.libsodium.org/doc/)
+* [Openssl](https://www.openssl.org)
+* gmp
+
+See the documentation [here](https://github.com/aeternity/aeternity/blob/master/README.md) detailed instructions for manual installation.
```
Comment (venimus) [inline README.md]: This sentence seem is not grammatically correct.

File: README.md (hunk 4 above)
Comment (venimus) [inline README.md]:
```suggestion
Clone the [git repository](https://github.com/aeternity/installer) and run:
```

File: README.md [hunk 6]
```diff
@@ -5,16 +5,57 @@ Run below command to install latest version of aeternity node:
...
 To install an older specific version use:
 ```bash
 bash <(curl -s https://install.aeternity.io/install.sh) 4.2.1
 ```
 
-See the documentation [here](https://github.com/aeternity/aeternity/blob/master/README.md) detailed instructions for manual installation.
+##### Additional options
+
+- `--no-prompt` - Disable confirmation prompt
+- `--delete` - Deleting the directory where aeternity node will be installed
```
Comment (venimus) [inline README.md]:
```suggestion
- `--delete` - Clean the target directory before installation
```

File: README.md [hunk 7]
```diff
@@ -5,16 +5,57 @@ Run below command to install latest version of aeternity node:
 bash <(curl -s https://install.aeternity.io/install.sh)
 ```
 
+The latest argument is interpreted as version.
```
Comment (venimus) [inline README.md]:
```suggestion
The last argument is interpreted as a version.
```

Comment (IvayloPapazov) [reply README.md]: 9f52faf

File: README.md (hunk 6 above)
Comment (IvayloPapazov) [reply README.md]: 9f52faf

File: README.md (hunk 4 above)
Comment (IvayloPapazov) [reply README.md]: 9f52faf

File: README.md (hunk 3 above)
Comment (IvayloPapazov) [reply README.md]: 9f52faf

**Review by dincho — DISMISSED** [review]

File: README.md (hunk 5 above)
Comment (venimus) [reply README.md]: `For manual installation see the [detailed instructions](https://github.com/aeternity/aeternity/blob/master/README.md).`

Comment (IvayloPapazov) [reply README.md]: a7b4ad4

**Review by venimus — APPROVED** [review]

**Review by dincho — APPROVED** [review]

**Pull request merged**